docs/en/compute/Spark.md

```diff
@@ -113,7 +118,7 @@ Also, the input file `Input_HDFS` now will be 100% loaded in the Alluxio file sy
 
 When connecting to the Alluxio HA cluster using internal leader election,
 add the following lines to `${SPARK_HOME}/conf/spark-defaults.conf` so Spark applications
-can know the Alluxio masters to connect to and find out the leader master.
+know which Alluxio masters to connect and which is the leader master.
```

Comment:
why remove the "to" after "connect"? nit: change to "connect to and identify the leader."?